File: LC1822.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        count = 0

        for i in nums:
            if i == 0:
                return 0
            if i < 0:
                count += 1
            
        return 1 if count%2 == 0 else -1

File: LC1822.cpp
class Solution {
public:
    int arraySign(vector<int>& nums) {
        int count = 0;

        for(auto i :nums){
            if(i == 0)
                return 0;
            
            if(i < 0)
                count++;
        }
    
        return count%2 == 0? 1 : -1;
    }
};
